install/helm/open-match/values.yaml

```diff
@@ -19,35 +19,35 @@
 swaggerui: &swaggerui
   hostName: om-swaggerui
   httpPort: 51500
-  portType: LoadBalancer
+  portType: ClusterIP
   replicas: 3
   image: openmatch-swaggerui
 mmlogic: &mmlogic
   hostName: om-mmlogic
   grpcPort: 50503
   httpPort: 51503
-  portType: LoadBalancer
```

Comment:
in the Helm chart documentation, did we specify how to actually switch between these port types? I do see folks needing the LB type in their E2E and would prefer not requiring the understanding of Helm chart / subchart value coalescing to achieve that.

Reply:
Yup, #713 already covered this part.